packages/components/src/Menu/MenuItem.tsx

```diff
@@ -183,6 +198,15 @@ const Detail = styled.div`
   padding-left: ${({ theme: { space } }) => space.large};
 `
 
+export const MenuItem = styled(MenuItemInternal)`
+  display: flex;
+  ${({ description, theme: { space } }) =>
+    description &&
+    `
+      padding-bottom: ${space.small};
+    `}
+`
```

Comment:
...and revert this as well (see comments above).
```suggestion
export const MenuItem = styled(MenuItemInternal)``
```